man/en/sos.1

```diff
@@ -75,6 +75,14 @@ commands and components than is directly available in either other manpages or
 
 See \fB sos help --help\fR and \fB man sos-help\fR for more information.
 
+.TP
+.B upload
+This subcommand takes as input a file to be uploaded to a distribution-defined
+target, or a user-defined one. The file can be an SoS archive or any other
+type of file like a vmcore, logfile, etc.
+
```

Comment:
Definitely not a native English speaker here :), but I suggest one re-wording and some subtle changes from DeepL:

    This subcommand uploads an input file to either a distribution-defined or a user-defined target. The file can be an SOS archive or any other type of file, such as a vmcore or log file.